curation-migration-datasync/index.ts

```diff
@@ -41,8 +49,16 @@ export async function handlerFn(event: SQSEvent): Promise<SQSBatchResponse> {
  * actually performs the logic for processing the event.
  */
 async function _handlerFn(
-  eventBody: EventBridgeEvent<EventDetailType, ScheduledItemPayload>
+  eventBody: EventBridgeEvent<EventDetailType, any>
```

Comment:
tried `ScheduledItemPayload | ApprovedItemPayload` but it doesn't work, so changed it to `any`